src/Styleguide/Components/Tabs.tsx

```diff
@@ -55,7 +55,8 @@ export class Tabs extends React.Component<TabsProps, TabsState> {
   constructor(props) {
     super(props)
 
-    const activeTabIndex = props.initialTabIndex || 0
+    const activeTabIndex =
+      props.initialTabIndex || this.props.children.findIndex(tab => tab != null)
```

Comment:
Couldn't it be the case that `tab` is reported as `false` (in the case when you don't want to render a certain tab) and thereby causing this logic to report a `tab` that's not really a `tab`?